### PR TITLE
Feature/button customisation

### PR DESCRIPTION
**Pull request by ArnoFleckenstein**
Buttons changed from bootstrap green to bootstrap blue. Button texts standardised. And buttons without bootstrap changed to bootstrap buttons.

### CODE REVIEW

**Comment by astroidex** [comment]
Backend please check data source -> WMS  -> delete

Bildschirmfoto vom 2024-03-18 14-58-49

**Comment by astroidex** [comment]
Backend add space between buttons

Bildschirmfoto vom 2024-03-18 15-01-57

File: src/Mapbender/CoreBundle/Resources/translations/messages.fr.yaml [hunk 1]
```diff
@@ -83,6 +83,7 @@ mb:
         description: Permet de créer un POI (point d'intérêt) pour le partager
       label:
         text: Text
+      accept: 'créer'
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.fr.yaml:86]: upper case

File: src/Mapbender/PrintBundle/Resources/translations/messages.de.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ mb:
       popup:
         btn:
           ok: Exportieren
-          cancel: Abbrechen
+          cancel: Schließen
```
[the remark this answers is not in the document]
Comment (ArnoFleckenstein) [reply src/Mapbender/PrintBundle/Resources/translations/messages.de.yaml:7]: I have no idea why I changed that. It is not used. But now it is correct. But it is still not used.

File: src/Mapbender/CoreBundle/Resources/translations/messages.fr.yaml (hunk 1 above)
Comment (ArnoFleckenstein) [reply src/Mapbender/CoreBundle/Resources/translations/messages.fr.yaml]: Has already been changed with the text correction "erstellen" to "Link erstellen".

**Pull request merged**